apps/barista-design-system/src/components/ba-decision-graph/ba-decision-graph-node/ba-decision-graph-node.scss

```
@@ -0,0 +1,144 @@
+@import 'interactive-common';
+@import 'colors';
+@import 'font-mixins';
+
+:host {
+  max-width: 100%;
+}
+
+::ng-deep button {
```

Please prefix `::ng-deep` with a `:host` selector. ng-deep lets you escape the ViewEncapsulation and without this preset, this rule will be applied on a document level (which in this case means all buttons would get this rule applied).

```suggestion
:host ::ng-deep button {
```